custom/conf/app.example.ini

```diff
@@ -1070,6 +1070,8 @@ ROUTER = console
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; List of reasons why a Pull Request or Issue can be locked
 ;LOCK_REASONS = Too heated,Off-topic,Resolved,Spam
+;; Maximum number of pinned Issues
+;MaxPinned = 6
```

Comment:
That's not the style of other config options 😀

Reply:
Fixed